### PR TITLE
Adding features of IS 5.1.0

### PR DESCRIPTION
**Pull request by cdwijayarathna**

### CODE REVIEW

**Comment by hasinthaindrajee** [comment]
Do not merge this. Will update PR once ready to merge

File: pom.xml [hunk 1]
```diff
@@ -949,7 +950,300 @@
...
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.xacml.server.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.xacml.ui.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
                                 <featureArtifactDef>
                                     org.wso2.carbon.mediation:org.apache.synapse.transport.fix.feature:${carbon.mediation.version.4.4.10}
                                 </featureArtifactDef>
```
Comment (Nipuni) [inline pom.xml:1251]: Please re order the features in alphabetical order

File: pom.xml [hunk 2]
```diff
@@ -833,37 +835,115 @@
...
                                     org.wso2.carbon.identity:org.wso2.carbon.identity.application.authenticator.basicauth.server.feature:${carbon.identity.version.4.5.6}
                                 </featureArtifactDef>
                                 <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.application.authenticator.basicauth.server.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.application.authenticator.iwa.server.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.application.authenticator.fido.server.feature:${carbon.identity.version.5.0.7}
```
Comment (Nipuni) [inline pom.xml]: org.wso2.carbon.identity.application.authenticator.iwa.server.feature and org.wso2.carbon.identity.application.authenticator.fido.server.feature are not in order

File: pom.xml [hunk 3]
```diff
@@ -1528,6 +1824,55 @@
...
                                 </featureArtifactDef>
 
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.agents.category.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.authorization.category.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.authenticators.category.feature:${carbon.identity.version.5.0.7}
```
Comment (Nipuni) [inline pom.xml]: org.wso2.carbon.identity.authorization.category.feature and org.wso2.carbon.identity.authenticators.category.feature are out of order

File: pom.xml [hunk 4]
```diff
@@ -1630,6 +1975,222 @@
...
+                                            <id>org.wso2.carbon.identity.authorization.category</id>
+                                            <version>${carbon.identity.version.5.0.7}</version>
+                                        </catFeature>
+                                    </features>
+                                </category>
+
+                                <!--Carbon authenticators category-->
+                                <category>
+                                    <id>org.wso2.carbon.identity.authenticators.category.feature.category</id>
+                                    <label>Carbon Authenticators</label>
```
Comment (Nipuni) [inline pom.xml]: org.wso2.carbon.identity.authorization.category.feature.category and org.wso2.carbon.identity.authenticators.category.feature.category are not in order

**Comment by hasinthaindrajee** [comment]
Please merge this pull request and release

**Pull request merged**